### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by kilpert**
Fix some bugs

### CODE REVIEW

**Comment by kilpert** [comment]
The fixed workflow runs fine for me. Maybe the test is broken?

File: workflow/rules/trim.smk [hunk 1]
```diff
@@ -32,7 +32,7 @@ rule cutadapt_pe:
     log:
         "logs/cutadapt/{sample}-{unit}.log",
     params:
-        others=config["params"]["cutadapt-pe"],
+        extra=config["params"]["cutadapt-pe"],
```
Comment (dlaehnemann) [inline workflow/rules/trim.smk:35]: Very important fix, to ensure the `params` from the `config.yaml` are actually picked up!

**Review by dlaehnemann — CHANGES_REQUESTED** [review]
@kilpert Many thanks for these fixes. I only saw them now, while wrapping up another PR. Would you be willing to address my suggestions and rebase onto the latest master? Then we could get your fixes in!

**Comment by dlaehnemann** [comment]
Almost all of the suggested changes have been incorporated through the recent pull requests. For example:
#61

The ones that weren't incorporated yet, I have added in PR #66 currently under review (referencing the commits in this PR).

So many thanks for these useful suggestions. I'll close this PR for now, as all of its suggestions have found their way into the code base.

**Pull request closed without merging**